include/z64play.h

```diff
@@ -0,0 +1,555 @@
+#ifndef Z64PLAY_H
+#define Z64PLAY_H
+
+#include "ultra64.h"
+#include "z64state.h"
+#include "z64actor.h"
+#include "z64animation.h"
+#include "z64collision_check.h"
+#include "z64cutscene.h"
+#include "z64environment.h"
+#include "z64light.h"
+#include "z64math.h"
+#include "z64message.h"
+#include "z64object.h"
+#include "z64sram.h"
+#include "z64skybox.h"
+#include "z64scene.h"
```

Comment:
```suggestion
#include "z64actor.h"
#include "z64animation.h"
#include "z64collision_check.h"
#include "z64cutscene.h"
#include "z64environment.h"
#include "z64light.h"
#include "z64math.h"
#include "z64message.h"
#include "z64object.h"
#include "z64sram.h"
#include "z64skybox.h"
#include "z64scene.h"
#include "z64state.h"
```

Reply:
The order is intentional, state is a "upper" system and others are "lower"

Not that it matters too much though idk

Reply:
Hmm, if they are intentional than that is fine. Though what do you mean by "upper" and "lower"?

Reply:
I mean hierarchically, PlayState inherits from the State system, and has the other Contexts as children

to rephrase, Play works inside the State system, the other systems in here work inside the Play system

Idk maybe it only makes sense to me? 😅

Reply:
I ended up reverting making a header for play. I think it was a bit of a premature move given it can/should be split further